internal/server/errors.go

```diff
@@ -56,7 +57,9 @@ func (a *API) sendAPIError(c *gin.Context, err error) {
 	case errors.Is(err, (*validator.InvalidValidationError)(nil)):
 		resp.Code = http.StatusBadRequest
 		resp.Message = err.Error()
-
+	case errors.Is(err, context.DeadlineExceeded):
+		resp.Code = http.StatusBadGateway // not ideal, but StatusRequestTimeout isn't intended for this.
```

Comment:
Maybe [504 Gateway Timeout](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504) ?

Reply:
yeah. hard to know what's good here. Gateway timeout seemed a lot like we were pointing the finger at another service, but it's probably a timeout between us and the DB

Reply:
Ya, none of them seem like a perfect fit. "Gateway Timeout" is nice in that at least it mentions timeout.

"Bad Gateway" is a risk because that's what we special case that for "there was a problem talking to the identity provider". So we probably don't want to use it again for this.

Reply:
I actually thought I selected GatewayTimeout 🤣